tests/resources/build_module_test.go

```diff
@@ -0,0 +1,27 @@
+package tests
+
+import (
+	"testing"
+
+	cleanup "github.com/rancher/tfp-automation/framework/cleanup"
+	"github.com/rancher/tfp-automation/tests/extensions/provisioning"
+	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/suite"
+)
+
+type BuildModuleTestSuite struct {
+	suite.Suite
+}
+
+func (r *BuildModuleTestSuite) TestBuildModule() error {
```

Comment:
what is buildModule really testing? 

We should not return anything for this function, since it is a Test*

Reply:
@Josh-Diamond can further clarify that.

Reply:
Resolved offline in discussion